PatientRegistrationApp/src/components/SinglePatientFollowUp.vue

```diff
@@ -1,53 +1,52 @@
 <template>
 	<v-container fluid>
-
     <v-row align="center" justify="start">
       <v-col cols="12">
-      <div class="font-weight-regular">Use this QR code to easily check-in at the site where you receive your vaccine. This QR code contains an encrypted patient identifier so we can quickly and securely identify you and retrieve your information.</div></v-col></v-row> 
+        <div class="font-weight-regular">Use this QR code to easily check-in at the site where you receive your vaccine. This QR code contains an encrypted patient identifier so we can quickly and securely identify you and retrieve your information.</div>
+      </v-col>
+    </v-row>
     <v-row justify="center">
       <div>
 				<vue-qrcode
-					v-bind:value="qrValue"
-					v-bind:errorCorrectionLevel="correctionLevel" />
-			</div></v-row>
-
+          id="qrCodeId"
+          v-bind:value="qrValue"
+					v-bind:errorCorrectionLevel="correctionLevel"
+          />
+			</div>
+    </v-row>
      <v-row justify="center">
       <div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataPersonalInfo.familyName}}, 
 					{{dataPersonalInfo.givenName}} {{dataPersonalInfo.middleName}} {{dataPersonalInfo.suffix}}</span></div>
     </v-row>
-
-
     <v-row justify="center">
-      <v-btn color="secondary" class="ma-2 white--text">
-								Download
+      <v-btn color="secondary" class="ma-2 white--text" @click="generatePdf">
+				Download As PDF
 			</v-btn>
     </v-row>
-
-<v-row justify="center">
-  <div class="font-weight-medium"><br><br>How do you want to receive this QR code?</div>
-</v-row>
-<v-row justify="center">
-  <v-col cols="8" sm="4" md="3" lg="3">
-  <v-radio-group class="font-weight-medium">
-      <v-radio label="E-mail"></v-radio>
-      <v-radio label="SMS Message"></v-radio>
-      <v-radio label="Both"></v-radio>
-  </v-radio-group>
-   </v-col>
-</v-row>
-
+    <v-row justify="center">
+      <div class="font-weight-medium"><br><br>How do you want to receive this QR code?</div>
+    </v-row>
+    <v-row justify="center">
+      <v-col cols="8" sm="4" md="3" lg="3">
+        <v-radio-group class="font-weight-medium">
+            <v-radio label="E-mail"></v-radio>
+            <v-radio label="SMS Message"></v-radio>
+            <v-radio label="Both"></v-radio>
+        </v-radio-group>
```

Comment:
Can you bind a variable to this selection so that the choice is stored somewhere?